vertx-web/src/main/java/io/vertx/ext/web/RoutingContext.java

```diff
@@ -393,20 +409,47 @@ default void reroute(String path) {
    * index on the original list. For example if a user has en-US and en-GB with same quality and this order the best
    * match will be en-US because it was declared as first entry by the client.
    *
+   * @deprecated Use {@link #acceptableLanguages()} or {@link #parsedHeaders()}.{@link ParsedHeaderValues#acceptLanguage()}
    * @return the best matched locale for the request
    */
-  @CacheReturn
```

Comment:
Oh? `@CacheReturn` is not just some metadata for code readers? I didn't know that!